app/models/service_template_ansible_playbook.rb

```diff
@@ -138,22 +143,44 @@ def job_template(action)
     resource_actions.find_by!(:action => action.to_s.capitalize).configuration_template
   end
 
+  def new_dialogs(config_info)
+    [:provision, :retirement, :reconfigure].each do |action|
+      info = config_info[action]
+      if new_dialog_required(info)
+        info[:dialog_id] = create_new_dialog(info[:new_dialog_name], job_template(action), info[:hosts]).id
+      end
+    end
+  end
+
+  def new_job_templates(config_info, name, description, auth_user)
+    config_info.deep_merge!(send(:create_job_template, name, description, config_info, auth_user))
+  end
+
+  def job_template_modifications(name, description, config_info, auth_user)
+    [:provision, :retirement, :reconfigure].each do |action|
+      info = config_info[action]
+      if info && info.key?(:playbook_id) && !info.key?(:create_only)
```

Comment:
```
if job_template(action)
```
no need to add `:create_only`, but this method needs to be called before creating new job templates.